<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab3ControlPannel</class>
 <widget class="QWidget" name="Tab3ControlPannel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pPBRed">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IotQtClient.qrc">
     <normaloff>:/IotQtClient/Image/red_off.png</normaloff>
     <normalon>:/IotQtClient/Image/red_on.png</normalon>:/IotQtClient/Image/red_off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pPBBlue">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IotQtClient.qrc">
     <normaloff>:/IotQtClient/Image/blue_off.png</normaloff>
     <normalon>:/IotQtClient/Image/blue_on.png</normalon>:/IotQtClient/Image/blue_off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pPBPurple">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IotQtClient.qrc">
     <normaloff>:/IotQtClient/Image/purple_off.png</normaloff>
     <normalon>:/IotQtClient/Image/purple_on.png</normalon>:/IotQtClient/Image/purple_off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>230</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="pLEName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pPBName">
      <property name="text">
       <string>조회</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="IotQtClient.qrc"/>
 </resources>
 <connections/>
</ui>
